data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		conversation
+			`On approach to the spaceport on Earth, you request to speak to an official of significant standing on diplomatic business. Thanks to your renown from the Pug war, you are able to shortcut objections stemming from you yourself not being a diplomat. The diplomatic ID from the box Sayari gave you doesn't hurt either.`
+			`	The Republic Deputy Minister for Foreign Affairs meets you in a private room at the spaceport.`
+			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake. "Xilin Yang. It's an honor to meet a hero of the Pug crisis such as yourself."`
+			choice
+				`	"It was hard-fought, but it was necessary."`
+				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
+					goto humble
+			`	You play the part and try to act noble. Mr. Yang seems more than happy to embody respect for the hero of his preferred narrative.`
+			`	"You're every inch what I expected," he assures you before moving to take his seat.`
+				goto sit
+			label humble
+			`	You shrug, palms up, and explain, "You should listen to the xenoanthropologists. The Pug were just here to test us, to make us really consider what mattered and come together."`
+			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet."`
+			`	He offers you a genuine smile as he takes his seat.`
+			label sit
+			`	You sit down opposite him and explain how the Hai have proposed that you be their official contact point for humanity on account of your fame from the war.`
+			`	He asks a few questions for clarity, and when you're done he requests you wait while he makes a call. He leaves the room for a long while, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that most of humanity's food styles originated here.`
+			`	Eventually Mr. Yang returns. "So sorry for that. I hope you were looked after suitably while you waited."`
+			`	Once you nod your assent, he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official "line" is that the government does not trade in rumors. However, an appropriate response is clearly needed. Republic Intelligence briefed the Cabinet of the Free Worlds Senate just yesterday. They have demanded that they have a diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you to meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`
```

Comment:
Referring to previous comment: `the Deep is democratically and voluntarily a part of the republic that adopts Republic foreign policy and only seems to care about autonomy in internal affairs.`

If this changes in other campaigns then it would naturally get branches here when they happen.